proto/cosmos/crypto/secp256r1/keys.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package cosmos.crypto.secp256r1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/secp256r1";
+option (gogoproto.messagename_all) = true;
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.goproto_getters_all) = false;
+
+// PubKey defines a secp256r1 ECDSA public key.
+message PubKey {
+  // Point on secp256r1 curve in a compressed representation as specified in section
+  // 4.3.6 of ANSI X9.62.
+  bytes key = 1 [(gogoproto.customtype) = "ecdsaPK"];
+}
+
+// PrivKey defines a secp256r1 ECDSA private key.
+message PrivKey {
```

Comment:
Do we even need a PrivKey in protobuf? This should not be created in our keyring but just by hardware devices.

Reply:
I created it for completeness. Do you prefer to remove it?

Reply:
Hmm... I don't have a strong preference, but generally prefer to not include things which won't/shouldn't be used. For tests we have an internal privkey to use already right?

Reply:
yes, tests and private key generation will work with the `internal/ecdsa`.

Reply:
OK, so I will remove the PrivKey proto.

Reply:
`sdkcrypto.PrivKey` interface requires to be `proto.Message`. While it doesn't need to implement Marsha and Unmarshal method, it's meant to be used in protobuf ecosystem as well as being marshaled (eg `proto.Marshal(pb proto.Message)`. Hence I think it's better to keep the proto file and generated files. What do you think?